plugin/adsk/scripts/mayaUsdMenu.py

```diff
@@ -167,4 +212,108 @@ def aeAttributesMenuCallback(aeAttributeMenu):
 
         for _, schemaTypeName in schemaNames:
             isMultiApply = schemas[schemaTypeName]
-            _createSchemaMenuItem(pluginMenu, pluginName, schemaTypeName, isMultiApply)
+            _createApplySchemaMenuItem(pluginMenu, pluginName, schemaTypeName, isMultiApply)
+
+
+############################################################################
+#
+# Remove schema
+
+def _removeSchemaMenuItemCallback(prettyTypeName, schemaTypeName, instanceName, menuItem):
+    '''
+    Remove the given schema from the prims in the current selection.
+    '''
+    primPaths = _getUsdPathsInSelection()
+    if not primPaths:
+        print('Cannot remove schema "%s": no USD prim are currently selected.' % schemaTypeName)
```

Comment:
Two questions. (1) should this string be in resource and (2) should use `cmds.warning` instead of print?